doc/api/timers.md

```diff
@@ -354,6 +354,19 @@ const timersPromises = require('timers/promises');
   * `signal` {AbortSignal} An optional `AbortSignal` that can be used to
     cancel the scheduled `Immediate`.
 
+### `timersPromises.setInterval([delay[, value[, options]]])`
+
+* `delay` {number} The number of milliseconds to wait between iterations.
+   **Default**: `1`.
+* `value` {any} A value with which the iterator returns.
```

Comment:
Why is `value` interesting? I don't think it should likely be a part of this API - whomever is consuming it can tack values along if they wish.

Reply:
This here was to match how `setTimeout(fn, after, value)` went to `setTimeout(after, value)`, not any specific meaning here

Reply:
I don't think this makes sense for intervals to be honest

Reply:
It can be useful for context tracking. I've used value in setInterval before.

Reply:
Mind sharing an example of how this will be used?

Reply:
@benjamingr It makes it possible to avoid having to create a closure?